src/portfolio/PortfolioTable.js

```diff
@@ -12,9 +13,27 @@ const PortfolioTable = ({compact, contracts, proposals, onViewDetails}) => {
 		<table>
 			<thead>
 				<tr>
-					<th>Ref.</th>
-					<th>Purchase</th>
-                    <th>Indicative</th>
+					<th>
+						<FormattedMessage
+							id="ref"
+							description=""
```

Comment:
Is id mandatory? And not sure if we want to add description here... if it is empty we don't, if it isn't... need to think it through ;)